bulbs/contributions/views.py

```diff
@@ -101,12 +101,33 @@ def get_queryset(self):
         )
 
         if "start" in self.request.GET:
-            start_date = dateparse.parse_date(self.request.GET["start"])
+            start_param = self.request.GET["start"]
```

Comment:
Are these two blocks nearly identical? Can they be broken out into a shared nested helper def?